deploy/kubernetes/examples/demo-snapshotclass-using-secret-from-configuration.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  name: demo-snapshotclass
```

Comment:
should we use a unique name for each class yaml?

Reply:
what are you suggesting for the name?

Reply:
depends on its final filename

Reply:
and regardless of this ticket:
I also noticed with have a somewhat similar issue with the example statefulset names.
I think we should consider removing the non-combined statefulset yamls, and remove the "combined" from its name, to have just one, comprehensive statefulset.
what do you think?

Reply:
what final name? is there any conversation about it?

Reply:
and sure, sounds good, do you want me to change it in this ticket?

Reply:
yes, there is [a conversation about it](https://github.com/IBM/ibm-block-csi-driver/pull/353#discussion_r682488910).
if @zingero is ok with it, it's fine by me

Reply:
I'm ok with it

Reply:
done

Reply:
so could we have a [unique name](https://github.com/IBM/ibm-block-csi-driver/pull/353#discussion_r682489291) for each class?

Reply:
what are you suggesting for the name?

Reply:
same as the filename, minus the `.yaml`?

Reply:
do you think that if I am already here I should change all `snapshotclass` to `volumesnapshotclass`? there is no such thing as snapshotclass..

Reply:
I'm fine with either way.
@zingero?

Reply:
`demo-volumesnapshotclass-config-secret` is more precise than:
`demo-snapshotclass-config-secret`, but very long

Reply:
Long is better than short if we add precision or align the names or make it better to understand in lots of cases.
This is one of the cases IMO.

Reply:
done